gen3users/validation.py

```diff
@@ -565,3 +566,64 @@ def validate_users(user_yaml_dict, existing_resources):
             #     )
 
     return ok
+
+
+def validate_roles(user_yaml_dict):
+    """
+    Validates the "roles" section of the user.yaml by checking that all
+    required fields are present.
+
+    Args:
+        user_yaml_dict (dict): Contents of a user.yaml file.
+
+    Return:
+        ok(bool): whether the validation succeeded.
+    """
+    logger.info("- Validating roles")
+    ok = True
+    roles = user_yaml_dict["authz"].get("roles", [])
+    for role in roles:
+        role_id = role.get("id")
+        permissions = role.get("permissions")
+        ok = (
+            assert_and_log(
+                role_id is not None, "Role ID not specified in role {}".format(role)
```

Comment:
```suggestion
                role_id is not None, "'id' not specified in role {}".format(role)
```
knowing the exact name makes it easier to fix for people who are not familiar with the syntax :)
same for other logs